src/frontend/apps/lti_site/components/PublicVideoDashboard/PublicVODDashboard/index.tsx

```diff
@@ -20,6 +35,65 @@ export const PublicVODDashboard = ({ playerType }: PublicVODDashboardProps) => {
   const timedTextTracks = useTimedTextTrack((state) =>
     state.getTimedTextTracks(),
   );
+  const [showPanelTrigger, setShowPanelTrigger] = useState(true);
```

Comment:
I think all code added in this component should be added in a new component managing the case where we are in a live to VOD conversion. And you should mount this new component in the `PublicVideoDashboard` component by adding a new condition in it.